Source/WorkersES6/decodeDraco.js

```diff
@@ -332,7 +332,7 @@ function decodePrimitive(parameters) {
 }
 
 function decode(parameters) {
-  if (defined(parameters.primitive)) {
+  if (defined(parameters.bufferView)) {
```

Comment:
`decodeDraco` checks for the existence of a property in `parameters` to know whether it's a mesh or a point cloud. `primitive` wasn't a great choice to begin with because `primitive` isn't actually used in the worker, `bufferView` however is.